ste/xfer-anyToRemote.go

```diff
@@ -296,17 +296,27 @@ func epilogueWithCleanupSendToRemote(jptm IJobPartTransferMgr, s ISenderBase, si
 	}
 
 	s.Epilogue() // Perform service-specific cleanup before jptm cleanup. Some services may actually require setup to make the file actually appear.
-	
-	if info.S2SDestLengthValidation {
-		if s2sc, isS2SCopier := s.(s2sCopier); isS2SCopier { // TODO: Implement this for upload and download?
+
+	if info.DestLengthValidation {
+		if s2sc, isS2SCopier := s.(s2sCopier); isS2SCopier {
```

Comment:
This is coded as if the method is still defined only in the S2S copier interface.  But 
(a) The method is on all sender/uploaders now, so
(b) it should be defined on the base interface, ISenderBase, and not defined anymore on the S2S (derived) interface
(c) There should be no type assertion here, just use `s` since it will have the method to get the length